src/components/EyesOnMe.js

```diff
@@ -1 +1,7 @@
 // Code EyesOnMe Component Here
+
+const EyesOnMe = () => {
+  return <button onFocus={() => console.log('Good!')} onBlur={() => console.log('Hey! Eyes on me!')}>Eyes on me</button>
```

Comment:
It would be nice to move the anonymous functions in named functions here. This line is getting pretty long and hard to read.